harness/src/main/java/org/apache/geode/perftest/jvms/JVMLauncher.java

```diff
@@ -91,7 +91,7 @@ public void run() {
     command.add("-D" + RemoteJVMFactory.JVM_ID + "=" + jvmConfig.getId());
     command.add("-D" + RemoteJVMFactory.OUTPUT_DIR + "=" + jvmConfig.getOutputDir());
 
-    if (Boolean.getBoolean("withSsl")) {
+    if (jvmConfig.getJvmArgs().contains("-DwithSsl=true")) {
```

Comment:
This line was important. Changed it from looking at a system property to looking at the `TestConfiguration` set by the topology. This puts the setting (TLS) under the control of the topology. This means when you're benchmarking SNI you don't have to specify TLS—you automatically get TLS.